pkg/operatorsource/configuring.go

```diff
@@ -99,6 +106,17 @@ func (r *configuringReconciler) Reconcile(ctx context.Context, in *v1alpha1.Oper
 	return
 }
 
+// Set group, version, and kind strings from reference
+// Object the sdk client returns does not set these
+func ensureGVK(opsrc *v1alpha1.OperatorSource) {
```

Comment:
Suggestion: This seems to be acting on the member variables of `OperatorSource` type. This can be added as a member function of `OperatorSource`.

Reply:
That's a good idea

Reply:
Updated